_posts/2017-02-16-comm-call-v13.md

```diff
@@ -0,0 +1,50 @@
+---
+name: comm-call-v13
+layout: post
+title: Community Call v13 - How to ask questions so they get answered! Possibly by yourself!
+date: 2017-02-16
+authors:
+  - name: Stefanie Butland
+categories:
+  - blog
+tags:
+- ropensci
+- community
+- events
+---
+
+Our [Community Call on Tuesday, March 7th, 8-9 AM PST](https://github.com/ropensci/commcalls/issues/14), will cover "How to ask questions so they get answered! Possibly by yourself!". This one should satisfy both newer and more experienced users and developers.
+
```

Comment:
you asked about more text for intro, perhaps:

> Asking questions about programming so they can be answered is a skill you can develop - we're not just born with it.  The speakers will cover some of the skills you'll need.  Asking questions is important for all skill levels - no matter how good you are you'll always need to ask questions. 

i don't love the above paragraph, needs tweaking